hydrabus_framework/core/command/run.py

```diff
@@ -0,0 +1,20 @@
+import traceback
+from hydrabus_framework.core.logger import Logger
+
+
+def run_module(hbf_instance):
+    """
+    Check all arguments and run the selected module
+    :param current_module: Class, Current loaded module class
+    :param hbf_instance: hydrabus serial instance
+    :return:
+    """
+    logger = Logger()
```

Comment:
Logger should be instanciated only once and by the framework. Maybe check singleton pattern.

Reply:
Done [#10c3681](https://github.com/hydrabus-framework/framework/commit/10c368189272dfb79bbaec95279da987552428a0)